scripts/build_dists_archive.sh

```diff
@@ -50,8 +52,9 @@ if [ "$(uname)" == "Darwin" ]; then
   #system_profiler SPSoftwareDataType
   #sw_vers
   osversion=$(sw_vers -productVersion | cut -d '.' -f 1,2)
-
+  export OPENSSL_STATIC=1
 elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+  sDist="ubuntu"
```

Comment:
Doubling Envs - osname is already set to osx/ubuntu?

Reply:
Nope. osname is not strictly fixed to just those two values in the script.

Reply:
You only supporting two build options and that is where it is set, else it's going to break ... ie RedHat builds are going to break. I think it's wasteful and more to develop with ... You defaulting to OSX and the setting to Ubuntu in the Linux part, which is what ```$osname``` does, right above it? Your patch, just adding suggestion.

Reply:
Not a idea to check if the ```osname``` env is set to either of the supported options and exit with a warning if not?